tools/c7n_azure/c7n_azure/policy.py

```diff
@@ -168,6 +170,11 @@ def run(self, event=None, lambda_context=None):
         raise NotImplementedError("subclass responsibility")
 
     def provision(self):
+        if sys.version_info[0] < 3:
```

Comment:
isn't the focus of this branch to rectify this situation wrt to provisoning?

Reply:
It adds support for Windows/Mac and any version of 3 - but there was additional work to support 2 that can come later if we want 2 to work.